WalletWasabi.Fluent/Controls/PrivacyTextPresenter.cs

```diff
@@ -0,0 +1,74 @@
+using System.Linq;
+using System.Reactive.Linq;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Media;
+using Avalonia.Utilities;
+using WalletWasabi.Fluent.ViewModels;
+
+namespace WalletWasabi.Fluent.Controls;
+
+public class PrivacyTextPresenter : UserControl
+{
+	private GlyphRun? _glyphRun;
+	private double _width;
+	private FormattedText? _formattedText;
+
+	private FormattedText CreateFormattedText()
+	{
+		return new FormattedText(
+			"",
+			new Typeface(FontFamily, FontStyle, FontWeight),
+			FontSize,
+			TextAlignment.Left,
+			TextWrapping.NoWrap,
+			Size.Infinity);
+	}
+
+	private GlyphRun CreateGlyphRun(double width)
+	{
+		var privacyChar = UIConstants.PrivacyChar;
+
+		var glyphTypeface = new Typeface((FontFamily?) FontFamily).GlyphTypeface;
+		var glyph = glyphTypeface.GetGlyph(privacyChar);
+
+		var scale = FontSize / glyphTypeface.DesignEmHeight;
+		var advance = glyphTypeface.GetGlyphAdvance(glyph) * scale;
+
+		var count = (int) (width / advance);
```

Comment:
count can't be null otherwise we will get InvalidOperationException
https://github.com/AvaloniaUI/Avalonia/blob/401bc521d2947bf7699a530f3d6e43e7a32c6baf/src/Avalonia.Visuals/Media/GlyphRun.cs#L609

Reply:
there seems to be issue in  wallet info with text like '0' does not show any hashes

Reply:
maybe when if width < advance the count can get rounded down to zero

Reply:
[CreateGlyphRun] count='0', width='8', advance='8.4013671875', scale='0.0068359375', privacyChar='#'

Reply:
using System;

namespace ConsoleApp1
{
    class Program
    {
        static void Main(string[] args)
        {
			//[CreateGlyphRun] count='0', width='8', advance='8.4013671875', scale='0.0068359375', privacyChar='#'
			var width = 8.0;
			var advance = 8.4013671875;
			var count = (int) (width / advance);
			Console.WriteLine($"{count}");
        }
	}
}

yep 0

Reply:
math.ceiling is not good, as it can draw too much